contracts/RewardEscrowV2.sol

```diff
@@ -55,6 +55,9 @@ contract RewardEscrowV2 is
     /// @notice treasury address - this may change
     address public treasuryDAO;
 
+    /// @notice EarlyVestFeeDistributor address
+    address public earlyVestFeeDistributor;
```

Comment:
@tommyrharper If this is an upgrade, then this will brick the contract due to a storage collision

Reply:
Oh yeah damn, good catch! Totally missed that 😰